Remove docker statement

dedemorton opened the pull request:
Removed as requested in #4143 

@tsg Let me know if this statement needs to go somewhere else.

ruflin commented:
The note should go here: https://github.com/elastic/beats/blob/master/metricbeat/module/docker/_meta/docs.asciidoc

dedemorton commented:
@ruflin I've added the note to the docker section. I've changed the language because it sounded like *all* requests will fail if you specify a period that's less than 3. But I'm not sure that's true. If it is true, I can change the note to say, "Specifying less than 3 seconds will cause all requests to timeout, and no data will be reported." Just let me know.
